definitions/tools/star_align.cwl

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+cwlVersion: v1.0
+label: "STAR: align reads to transcriptome"
+baseCommand: STAR
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 32000
+      coresMin: 16
+    - class: DockerRequirement
```

Comment:
I'm surprised that this only takes 2G per core - have you run it enough to know that'll hold for most samples?